lib/src/Dropdown/Dropdown.tsx

```diff
@@ -27,25 +28,33 @@ export interface DropdownPlaceholders {
     noResultText?: string;
 }
 
-export interface DropdownProps {
+interface OverriddenNativeProps extends React.DetailedHTMLProps<React.SelectHTMLAttributes<HTMLSelectElement>, HTMLSelectElement> {
+    onChange: any;
+}
+
+export interface DropdownProps extends OverriddenNativeProps {
+    onChange: (value: DropdownValue) => void;
     className?: string;
     clearable?: boolean;
     disabled?: boolean;
     error?: string;
-    id?: string;
     label?: string;
     list: Array<DropdownItem>;
     more?: boolean;
+    /** @deprecated use `multiple` instead */
```

Comment:
we have quite a number of components with breaking changes, maybe we can remove this directly?

Reply:
yeah sounds good. I've added a todo in the PR for it.